simplesat/sat/policy.py

```diff
@@ -13,7 +20,99 @@ def get_next_package_id(self, assignments, clauses):
         """
 
 
+class PolicyLogger(IPolicy):
```

Comment:
Hm, that's a good point. Can you create a ticket related to this issue, and let's get back to it later while we've worked more on the `*Policy` classes.